pages/hub/lists/new.tsx

```diff
@@ -197,6 +253,24 @@ const CreateListPage = () => {
             }}
           />
 
+          <InfoCard
+            title="Import your GitHub team"
```

Comment:
```suggestion
            title="Sync your GitHub team"
```

Small copy change to "sync." We should be able to update the team if someone leaves.